src/app/communities/page.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
Why do we need it here?

Reply:
To use the context, there is another approach to use the [context in server components](https://vercel.com/guides/react-context-state-management-nextjs#rendering-third-party-context-providers-in-server-components) but I'm not sure if I should do it.